clar2wasm/src/standard/standard.wat

```diff
@@ -1468,6 +1465,40 @@
         (i32.store offset=28 (local.get $origin) (i32.add (i32.load offset=28 (local.get $origin)) (local.get $h)))
     )
 
+    ;;
+    ;; logical not implementation
+    ;;
+    (type $not-func (func (param $bool_in i32) (result i32)))
+    (func $not (type $not-func) (param $bool_in i32) (result i32)
+        (i32.eqz (local.get $bool_in))
+    )
+    (export "not" (func $not))
```

Comment:
I do like putting this right with the function, but the rest of the exports are all at the bottom. Let's move this one to the bottom for now, and maybe in a separate PR, we can move them all next to their functions, or better yet, figure out how to get the inline export working.

Reply:
Done